spec/requests/targets/index._spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'POST /targets', type: :request do
+
+  let!(:user)  { create(:user) }
+  let!(:target) { create(:target) }
+  let!(:targets) { create_list(:target, 3, user: user) }
+
+  subject { get targets_path, headers: auth_user_headers, as: :json }
+
+  it 'returns a successful response' do
+    subject
```

Comment:
Maybe you can do
```
before { subject }
```
instead of calling subject in every `it`

https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/before-and-after-hooks